src/core/best-practices.md

```diff
@@ -155,3 +155,50 @@ You can use our [Annotation Plugin](/core/software/pupil-capture/#annotations) t
 the recording on desired events, such as trigger events. Annotations can be sent via the keyboard or programmatically. 
 [This script](https://github.com/pupil-labs/pupil-helpers/blob/master/python/remote_annotations.py) demonstrates how you 
 can send remote annotations over the network.
+
+## Pupillometry
+[Pupillometry](https://doi.org/10.1002/wcs.1323) is the study of temporal changes in pupil diameter in response to 
+external light stimuli and/or cognitive processing. Pupil Core reports pupil diameter in mm provided by the 
+[pye3d](/developer/core/pye3d/#pye3d-pupil-detection) model: `diameter_3d`, and in pixels as observed in the eye 
+videos: `diameter`. 
+
+Pupil size in pixels is dependent on the eye-camera to pupil distance and is not corrected for perspective. 
+Pye3d, on the other hand, accounts for differences in eye-camera to pupil distances and corrects for perspective. 
+It thus more accurately reflects pupil size and will be preferable for most users.
+
+### pye3d model
+A well-fit [pye3d](/developer/core/pye3d/#pye3d-pupil-detection) model is important for accurate estimates of pupil size 
+in mm. To generate a well-fitting model, sample sufficient gaze points from a variety of gaze angles, e.g. by moving the 
+head around while looking at a fixed position. A well-fit model is visualized by a stable circle that surrounds the 
+modelled eyeball, and this should be of an equivalent size to the respective eyeball. A dark blue circle indicates that 
+the model is within physiological bounds, and a light blue circle out of physiological bounds.
+
+<v-img :src="require('../media/core/imgs/bp-pye3d.png')"></v-img>
+<br>
+
+See the [pye3d release notes](https://github.com/pupil-labs/pupil/releases/tag/v3.4)for further details.
+
+### Freeze the pye3d model
+:::tip <v-icon large color="info">info_outline</v-icon>
+A frozen model **is not** robust to headset slippage (erroneous movements of the headset on the wearer) as it is 
+prevented from adapting to headset movements. If using a frozen model, and the headset slips during the experiment, 
+erroneous pupil size estimates can occur. Therefore, only use this option if the experiment is tightly controlled with 
+limited head movements, and keep recordings short and free of slippage as much as possible.
+:::
+
+The [pye3d model](/developer/core/pye3d/#pye3d-pupil-detection) regularly updates to account for headset slippage. In certain situations, this can lead to incorrect pupil size 
+estimates:
+1. Hard Slippage: The headset slips on the wearer, and a 2d pupil datum is generated **prior** to the 3d model adjusting
```

Comment:
```suggestion
1. Large Headset Slippage: The headset slips on the wearer, and a 2d pupil datum is generated **prior** to the 3d model adjusting
```

Reply:
Hard is a strange word in this context IMO.

Reply:
The spatial amount of slippage is one thing. The temporal aspect needs also to be considered. Large slippage within short time is more difficult to compensate for the model compared to slippage over a longer period of time.

Reply:
I think we need to emphasise that 'Actual Headset Slippage' can be a cause of erroneous pupil size estimates, in addition to 'Model Adjustment Error' (where no headset slippage occurs). I don't think further details of the kind of slippage are needed. Therefore I have changed from 'Hard' to 'Actual'